kafka/src/main/java/com/metamx/tranquility/kafka/util/IngestionSpecHelper.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.metamx.tranquility.kafka.util;
+
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Charsets;
+import com.metamx.common.IAE;
+import com.metamx.tranquility.kafka.model.IngestionSpec;
+import com.metamx.tranquility.kafka.model.TranquilityConfig;
+import io.druid.jackson.AggregatorsModule;
+import io.druid.jackson.QueryGranularityModule;
+import io.druid.query.aggregation.histogram.ApproximateHistogramDruidModule;
+import org.apache.commons.lang.Validate;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Read and validate ingestion spec files.
+ */
+public class IngestionSpecHelper
+{
+  private static final ObjectMapper mapper = new ObjectMapper();
```

Comment:
This should be a `DruidGuicer.objectMapper` to make sure we load all extensions properly. That will also get the approximate histogram aggregator loaded if it's present, so the explicit dependency and code for that is not necessary.